### DISCUSSION TITLE
Has anyone got their sample wheel of fortune automations to work?

### DISCUSSION THREAD

**Discussion by AndrewNalty** [BottlecapDave/HomeAssistant-OctopusEnergy#610] [Q&A]
I've used the example automation and get this

Executed: 5 December 2023 at 20:12:23
Error: UndefinedError: 'dict object' has no attribute 'entity_id'

This is now beyond my capabilities...

**Comment by BottlecapDave** [comment]
Hello, sorry for the late response. I've created some blueprints, including this. You can find them at https://bottlecapdave.github.io/HomeAssistant-OctopusEnergy/blueprints/#wheel-of-fortune

**Reply by AndrewNalty** [reply]
What a shame - I was really hoping i could automate my 4p/month! Thanks for an amazing integration

**Reply by noisytwit** [reply]
Hey BCD, I have been using this blueprint for a couple of days and it works amazingly well for signing up for the saving Sessions automatically. Just wanted to say thanks as it saved me a load of headaches getting my own up and running.

I am going to now tweak the notification code slightly to push the notification to my phone vice persistent but otherwise all is well.

**Reply by BottlecapDave** [reply]
Glad to hear it worked out. The blueprints are there because people were having issues with the examples and are great for people that are new to HA. I hope that in the future blueprints have an easier way of remixing them (e.g. disconnecting them and having the ability to alter like any other automation)
